ISSUE_TEMPLATE.md

```diff
@@ -1,7 +1,18 @@
 Please **do not post any internal, closed source snippets** on this public issue tracker!
 
-### Expected behaviour
+### Description
 
-### Actual behaviour
+### Steps to Reproduce
+1. 
+2. 
+3. 
 
```

Comment:
I might suggest adding something like these additional sections, so that the user has somewhere to post the logs?
```
### Debug Output
<!--
Full debug output can be obtained by running Terraform with the environment variable `TF_LOG=trace`. Please create a GitHub Gist containing the debug output. Please do _not_ paste the debug output in the issue, since debug output is long.

Debug output may contain sensitive information. Please review it before posting publicly, and if you are concerned feel free to encrypt the files using the HashiCorp security public key.
-->

### Crash Output
<!--
If the console output indicates that Terraform crashed, please share a link to a GitHub Gist containing the output of the `crash.log` file.
-->
```
I've been referencing https://github.com/hashicorp/terraform/blob/ec90bed62d008b5c1d9ce291a03226633a7e3069/.github/ISSUE_TEMPLATE/bug_report.md as an example

Reply:
@cuminandpaprika I've created a pr related to this: #556